charter.md

```diff
@@ -4,6 +4,8 @@ The More Instant Messaging Interoperability (MIMI) working group will specify th
 
 Recognizing the need for a standardized security protocol to support group key establishment, authentication, and confidentiality services for messaging, the IETF has specified the Messaging Layer Security (MLS) protocol [I-D.ietf-mls-protocol] and architecture [I-D.ietf-mls-architecture]. MLS is agnostic to the identity system used within any given messaging service; it provides confidentiality of sessions once the participants in a conversation have been identified. To achieve interoperable messaging, the MIMI working group will specify how one or more identity building block technologies (for example, X.509 certificates or Verifiable Credentials) can be used to establish end-to-end cryptographic identity across messaging services, assuming the use of MLS for key establishment. 
 
+Interoperable messaging in federated environments requires consensus on a common delivery service and a transport protocol between federated domains. Specifically, the MLS protocol requires ordering of handshake messages within groups to ensure clients can synchronize despite asynchronous message delivery. This working group will specify a flexible solution for transport and delivery that takes into account typical requirements and best practices from the industry. 
```

Comment:
```suggestion
Interoperable messaging in federated environments requires consensus on a common delivery service and a transport protocol between federated domains. Specifically, the MLS protocol requires ordering of handshake messages within groups to ensure clients can synchronize despite asynchronous message delivery. This working group will specify a flexible solution for transport and delivery that avoids centralised control over conversations and takes into account typical requirements and best practices from the industry. 
```

We're very concerned on the Matrix side that...

> Specifically, the MLS protocol requires ordering of handshake messages within groups to ensure clients can synchronize despite asynchronous message delivery. 

...is a defect in MLS's design, and would like to ensure that MIMI does not suffer the same constraint of requiring a centralised sequencing server.  Therefore we'd like to propose that the much-needed transport/delivery solution avoids centralisation (while still building on MLS, of course)

Reply:
As discussed on the list, Raphael's proposed phrasing does not preclude fancy consensus mechanism so I think this is not needed.

Reply:
okay, agreed (as per BOF Q&A)